apps/linkedin-to-notion/src/contents/linkedin-notion-side-panel.tsx

```diff
@@ -32,6 +33,12 @@ const LinkedinNotionSidePanel = () => {
     }),
   });
 
+  const storeEntriesSecurely = async (key: string, value: object): Promise<void> => {
+    const storage = new SecureStorage({ area: 'local' });
+    await storage.setPassword('napoleon');
```

Comment:
I don't understand this line 🤔

Reply:
This function could be on a more global util file btw

Reply:
cf. longer comment below: let's use the hook you'd already put in the code to store the whole session

Reply:
For syntax matters: [read this](https://docs.plasmo.com/framework/storage#secure-storage).

For the intrinsic meaning of what I've done, here's my understanding: we're encrypting keys in the local storage, so that someone who does not have the password ('napoleon' in this case) won't be able to read what's in the storage EVER.

That being said, 'napoleon' is a hardcoded password. It's the same for everyone atm, and anyone who sneaks in our public repo would easily read the storage. I'm not sure how, but I don't see the point of a secure storage if it wasn't possible somehow.

Anyway, for the whole thing to work, I think we should have one password per user. I'm not sure what to use though. I've seen other repos using uids generated elsewhere in their app or even a password entered manually by their users. 

*Do you have any idea of what we could use as a pwd?*
---
On the other matter (making the function a util in another file): I don't want to do it now if that's ok, bc we use the function only once atm.

Reply:
What I don't understand is that your storing securely something that is already store elsewhere so there's no additional security. Let's talk about this IRL because I don't understand sorry 😬